teamcity/datadog_checks/teamcity/data/conf.yaml.example

```diff
@@ -361,3 +361,36 @@ instances:
     ## This is useful for cluster-level checks.
     #
     # empty_default_hostname: false
+
+## Log Section
+##
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs.
+## service - optional - The name of the service that generates the log.
+##                      Overrides any `service` defined in the `init_config` section.
+## tags - optional - Add tags to the collected logs.
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+#
+# logs:
+#   - type: file
+#     path: /opt/teamcity/logs/teamcity-server.log
```

Comment:
It's maybe cleaner to have only one `path: /opt/teamcity/logs/teamcity-*.log` rule ?

Reply:
They might not care about many of them https://www.jetbrains.com/help/teamcity/teamcity-server-logs.html#General+Logging+Description

WDYT?